node-scripts/yt-description.js

```diff
@@ -313,18 +313,16 @@ function writeDescription(video) {
   }
 
   // Related Challenges
+
   if (data.relatedChallenges && data.relatedChallenges.length > 0) {
     description += `\nRelated Coding Challenges:\n`;
     for (const challenge of data.relatedChallenges) {
       const challengeData = videos.find(
         (vid) => vid.pageURL === `challenges/${challenge}`
       );
       if (challengeData) {
-        const {
-          videoNumber,
-          challengeTitle: title,
-          pageURL: url
-        } = challengeData.data;
+        const { videoNumber, title } = challengeData.data;
+        const url = challengeData.pageURL;
         description +=
           `🚂 #${videoNumber} ${title}: ${getYouTubeURL(url)}`.trim() + '\n';
```

Comment:
```suggestion
        const { videoNumber, challengeTitle } = challengeData.data;
        const url = challengeData.pageURL;
        description +=
          `🚂 #${videoNumber} ${challengeTitle}: ${getYouTubeURL(url)}`.trim() + '\n';
```

This fixes the annoying bug of showing `Asteroids - Part 1` for the title. 

This is a hacky fix for multipart challenges which I have created wherein the `title` also contains the Part information, but the `challengeTitle` is just the name of the challenge. This is required as in different places different versions are needed.